### DISCUSSION TITLE
Detached components keep zapping

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#14052] [Bug reports]
### Disclaimers  

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

if you disconnect a wire on one side and detach the component then disconnected wire will still be attached to it and keep zapping, which is annoying

https://github.com/FakeFishGames/Barotrauma/assets/122838333/0a9b5925-f179-4ce3-9c63-64d56cd17474

### Reproduction steps

connect 2 components with wire
detach wire on one side, but don't remove it
detach that component
wire still connected to that component

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I don't think this is actually a bug, it's intentional that the wire is left hanging like this if the other end is not connected to anything. However 

> This can also be used to magically transfer signals and power between different subs at any distances

This seems like a bug, could you elaborate on how you can use this to transfer signals and power?

**Reply by SomeRandomNoobKekeke** [reply]
Normally when you detach a component all connected wires drops, but if a wire was disconnected then it'll still be hanging on that component so you can place that component on the other sub, connect hanging wire back and it'll work

https://github.com/FakeFishGames/Barotrauma/assets/122838333/82dcc397-4699-4c01-9d7b-60d53119a90e

**Comment by Jasontti** [comment]
Tested the fix for wireless components and found no issues.
Closing this discussion as resolved.

**Reply by SomeRandomNoobKekeke** [reply]
My plans for making fleet of op, exploit based subs
![evaporate-disappear](https://github.com/FakeFishGames/Barotrauma/assets/122838333/1c58a1ec-3fe9-4e7c-9a2d-02bf0a57d196)
